shade-test/jar-file/src/test/java/org/testcontainers/JarFileShadingTest.java

```diff
@@ -0,0 +1,79 @@
+package org.testcontainers;
+
+import org.assertj.core.api.ListAssert;
+import org.junit.*;
+
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.*;
+
+import static java.util.Collections.emptyMap;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class JarFileShadingTest {
+
+    private static FileSystem fileSystem;
+
+    private static Path root;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        Path path = Paths.get("..", "..", "core", "target", "testcontainers-0-SNAPSHOT.jar");
```

Comment:
It's going to be pretty obvious why this has broken if someone ever manages to break it, so I think we'll be OK with this.